### ISSUE TITLE
Problem in login

### ISSUE THREAD

**Issue by saxenashobhit** [hybridauth/hybridauth#984]
Hi, 

I am applying hybridauth library to WordPress plugin. I am using Hybridauth 3.0.0-rc.6.
When I register to my site using google, I don't face any problem, the user easily registers to Wordpress Site and logged in to Wordpress as a user. But when the same user again login to Wordpress using Google authentication, after Google authentication user doesn't redirect to Wordpress dashboard it gives following messages:

**This page isn’t working
www.woow-themes.com redirected you too many times.
Try clearing your cookies.
ERR_TOO_MANY_REDIRECTS**

But when I tested it to localhost, I didn't face any issue. 

Please let me know.

Thanks.

**Comment by ApacheEx** [comment]
Hi,
is it still actual? It seems server issues, because you didn't get the error on your local machine.

**Comment by isreehari** [comment]
Yes. It is happening to me as well in while I am doing the google authentication.

**Comment by isreehari** [comment]
That's my bad, but my research information will help others. First thing I am using the codeigniter and it does not set the `filter_input(INPUT_GET, 'code')` by default I changed it to `filter_input(INPUT_GET, 'code') || ( isset($_GET) && $_GET['code'); `. Other thing is my session was unstable, it was creating the new session on redirect so there is no problem with hybridauth.

**Comment by Jay007123** [comment]
Im facing too many redirect issue CI ver.3.1.11
